Dockerfile

```diff
@@ -282,13 +283,20 @@ VOLUME /tmp
 ENV CGO_ENABLED=0
 # copy cosign binary for integration test
 COPY --from=gcr.io/projectsigstore/cosign:v2.0.0@sha256:728944a9542a7235b4358c4ab2bcea855840e9d4b9594febca5c2207f5da7f38 /ko-app/cosign /usr/local/bin/cosign
+# installing soci for integration test
+ARG SOCI_SNAPSHOTTER_VERSION
+https://github.com/awslabs/soci-snapshotter/releases/download/v0.3.0/soci-snapshotter-0.3.0-linux-amd64.tar.gz 
+RUN fname="soci-snapshotter-${SOCI_SNAPSHOTTER_VERSION}-${TARGETOS:-linux}-${TARGETARCH:-amd64}.tar.gz" && \
+  curl -o "${fname}" -fSL "https://github.com/awslabs/soci-snapshotter/releases/download/v0.3.0/soci-snapshotter-0.3.0-linux-amd64.tar.gz" && \
+  tar -C /usr/local/bin -xvf "${fname}" ./soci ./soci-snapshotter-grpc
```

Comment:
An interesting finding is the `tar` in nerdctl test-integration container doesn't properly support the format like `./soci` by some reason. Try to remove `./` for both.